app/Http/Requests/DailyExpenseRequest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class DailyExpenseRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
```

Comment:
While authorize will not be used to authorize users, we can totally remove it from the requests classes

Reply:
done